### ISSUE TITLE
[FEATURE REQUEST] Global option to disable setting up all tests

### ISSUE THREAD

**Issue by spezialspezial** [OpenImageIO/oiio#3189]
To speed up CMake configuration it would be nice to have a global option to skip setting up tests at all if no tests are built.

Solution might be to add a separate switch or checking OIIO_BUILD_TESTS value before adding the tests:

```
if (NOT ${PROJECT_NAME}_IS_SUBPROJECT AND OIIO_BUILD_TESTS)
    oiio_setup_test_data()
    oiio_add_all_tests()
endif ()

```

**Comment by lgritz** [comment]
Addressed by #3227

**Comment by spezialspezial** [comment]
Thanks, not against testing per se, oiio-testsuite is just quite the chubby beast.
